src/cursor.c

```diff
@@ -160,6 +160,7 @@ process_cursor_resize(struct server *server, uint32_t time)
 	else if (server->resize_edges & WLR_EDGE_RIGHT)
```

Comment:
I know this is not part of your changes but would you mind adding the curly braces here as well?